plugins/quota/actor.go

```diff
@@ -0,0 +1,87 @@
+package quota
+
+import (
+	"os"
+	"path"
+
+	"github.com/gluster/glusterd2/glusterd2/daemon"
+	"github.com/gluster/glusterd2/glusterd2/volume"
+	"github.com/gluster/glusterd2/glusterd2/xlator"
+	log "github.com/sirupsen/logrus"
+)
+
+type quotadActor struct{}
+
+const (
+	quotaDaemonKey = "enable"
+)
+
+// IsQuotadAffectedNode returns true if there are local bricks of volume on
+// which quota is enabled
+func isQuotadAffectedNode(logger log.FieldLogger) bool {
+	volumes, err := volume.GetVolumes()
```

Comment:
This error should be propagated back to caller. You can move `volume.GetVolumes()` call to the `Do()` and pass volumes to this function. In that case, you don't have to pass logger too.

Reply:
done.